src/test/resources/regressions/features/opaque/reveal-closure-fail1.gobra

```diff
@@ -0,0 +1,20 @@
+// Any copyright is dedicated to the Public Domain.
+// http://creativecommons.org/publicdomain/zero/1.0/
+
+package pkg
+
+func main() {
+	x@ := 0
+	c := preserves acc(&x)
+		 ensures x == old(x) + n && m == x
+		 func f(n int) (m int) {
+			x += n;
+			return x
+		 }
+
+	// QUES: Not sure how to test that we cannot reveal closure calls; the following
```

Comment:
How does a closure call work in Gobra?

Reply:
is `reveal (c(10) as f)` accepted?

Reply:
No, I get an error of the following form:
```
[info] Error at: <src/test/resources/regressions/features/opaque/reveal-closure-fail1.gobra:17:25> unexpected end of line, expecting arguments
[info] 	r := reveal (c(10) as f)
```
Dropping reveal allows the file to verify.

Reply:
This is a snippet from the parser:
```
  ...
  | primaryExpr arguments #invokePrimaryExpr
  | REVEAL primaryExpr arguments #revealInvokePrimaryExpr
  | primaryExpr arguments AS closureSpecInstance #invokePrimaryExprWithSpec
  ...
```
My guess is that since we do not have a rule like `REVEAL primaryExpr arguments AS closureSpecInstance #revealInvokePrimaryExprWithSpec` at the moment, trying to reveal closure calls doesn't even parse. Should I simply drop this test?